static/app/views/replays/detail/console/consoleMessage.tsx

```diff
@@ -123,8 +129,10 @@ const ConsoleTimestamp = styled(Common)<{isLast: boolean; level: string}>`
   cursor: pointer;
 `;
 
-const Icon = styled(Common)`
+const Icon = styled(Common)<{isActive: boolean}>`
   padding: ${space(0.5)} ${space(1)};
+  border-left: 4px solid
+    ${p => (p.isActive ? `${p.theme.purple300}` : `${p.theme.background}`)};
```

Comment:
Instead of directly using the colors, it's preferable to use an alias. So instead of `purple300`, we probably want something like `active` or `focus`.